content/docs/traffic-management/header-control/request-header.md

```diff
@@ -335,192 +148,119 @@ Add headers from a Kubernetes secret for requests to a specific route.
        - headers.example
      rules:
        - filters:
-           - type: ExtensionRef
-             extensionRef:
-               group: gateway.solo.io
-               kind: RouteOption
-               name: header-manipulation
+           - type: RequestHeaderModifier
+             requestHeaderModifier:
+               set: 
+               - name: my-header
+                 value: kgateway
          backendRefs:
            - name: httpbin
              port: 8000
    EOF
    ```
 
-4. Send a request to the httpbin app on the `headers.example` domain and verify that you get back a 200 HTTP response code and the headers that you added to the secret. 
-   * **LoadBalancer IP address or hostname**
-     ```sh
-     curl -vik http://$INGRESS_GW_ADDRESS:8080/headers -H "host: headers.example:8080"
-     ```
-   * **Port-forward for local testing**
-     ```sh
-     curl -vik localhost:8080/headers -H "host: headers.example"
-     ```
-
-   Example output: 
-   ```yaml {linenos=table,hl_lines=[22,23,24,25,26],linenostart=1}
-   * Mark bundle as not supporting multiuse
-   < HTTP/1.1 200 OK
-   HTTP/1.1 200 OK
-   ..
-   {
-      "headers": {
-        "Accept": [
-          "*/*"
-        ],
-        "Host": [
-          "headers.example:8080"
-        ],
-        "User-Agent": [
-          "curl/7.77.0"
-        ],
-        "X-Envoy-Expected-Rq-Timeout-Ms": [
-          "15000"
-        ],
-        "X-Forwarded-Proto": [
-          "http"
-        ],
-        "X-Header-1": [
-          "one"
-        ],
-        "X-Header-2": [
-          "two"
-        ],
-        "X-Request-Id": [
-          "a7b891d2-93a2-4029-bd98-cd46dd414d03"
-        ]
-      }
-   }
-   ...
-   ```
-
-5. Optional: Remove the resources that you created. 
-   ```sh
-   kubectl delete httproute httpbin-headers -n httpbin
-   kubectl delete routeoption header-manipulation -n httpbin
-   kubectl delete secret my-headers -n httpbin
-   ```
+   |Setting|Description|
+   |--|--|
+   |`spec.parentRefs`| The name and namespace of the gateway that services this HTTPRoute. In this example, you use the `http` gateway that was created as part of the get started guide. |
+   |`spec.rules.filters.type`| The type of filter that you want to apply to incoming requests. In this example, the `RequestHeaderModifier` filter is used.|
+   |`spec.rules.filters.requestHeaderModifier.set`|The name and value of the request header that you want to set. |
+   |`spec.rules.backendRefs`|The Kubernetes service you want to forward traffic to. In this example, all traffic is forwarded to the httpbin app that you set up as part of the get started guide. |
```

Comment:
Could it also be a Backend? maybe update to backend destination or something if so. Similar comments on the tables throughout.